src/windows/win-toggle-safe-mode.ps1

```diff
@@ -0,0 +1,166 @@
+#########################################################################################################
+<#
+# .SYNOPSIS
+#   Disable Safe Mode if your Windows VM is booting in Safe Mode. Also activates Safe Mode 
+#   if you require it (e.g. to uninstall certain software).
+#
+# .DESCRIPTION
+#   Azure VMs do not natively support Safe Mode because RDP access is disabled in Safe Mode. Some users
+#   need to boot their VM in Safe Mode for specific reasons (e.g. uninstalling certain software). Other
+#   users may find their VM booting into Safe Mode inadvertantly due to user error or misconfiguration,
+#   which will disable RDP access until corrected. This script utilizes the az vm repair extension to 
+#   clone the VM into a Hyper-V environment using Nested Virtualization and toggle Safe Boot. The user
+#   may then access their VM in Safe Mode via the Rescue VM or revert Safe Mode on their Azure VM. They 
+#   may then swap the disk using the `az vm repair restore` functionality.
+#
+#   Testing:
+#       1. Copied scripts to newly created Windows Server 2019 Datacenter (Gen 1)
+#       2. Ran win-enable-nested-hyperv once to install Hyper-V, restarted, and ran again to create new nested VM
+#       3. Ran win-toggle-safe-mode.ps1, worked successfully in toggling Safe Mode
+#       4. Set up new VM and ran the following from my local machine, worked successfully (~69 seconds): 
+#           az vm repair run -g sourcevm_group -n sourcevm --custom-script-file .\win-toggle-safe-mode.ps1 --verbose --run-on-repair
+#       5. Tried on a WS 2016 Gen 2 Azure VM, but was unsuccessful, not compatible with Gen 2 right now
+#       6. Tested on WS2012R2, WS2016 Datacenter, and WS2019 Datacenter (Gen 1)
+#
+#   https://docs.microsoft.com/en-us/cli/azure/ext/vm-repair/vm/repair?view=azure-cli-latest
+#   https://docs.microsoft.com/en-us/troubleshoot/azure/virtual-machines/troubleshoot-rdp-safe-mode
+#
+# .PARAMETER safeModeSwitch
+#   "On" to enable Safe Mode, "Off" to disable Safe Mode, no parameter to toggle whatever the current state is.
+#
+# .EXAMPLE
+#   az vm repair run -g sourceRG -n sourceVM --run-id win-toggle-safe-mode --verbose --run-on-repair
+#   az vm repair run -g sourceRG -n sourceVM --run-id win-toggle-safe-mode --parameters safeModeSwitch=on --verbose --run-on-repair
+#   az vm repair run -g sourceRG -n sourceVM --run-id win-toggle-safe-mode --parameters safeModeSwitch=off --verbose --run-on-repair
```

Comment:
Since [parameters are currently not working for custom scripts](https://github.com/Azure/repair-script-library/blob/master/doc/adding_new_scripts.md) I copied all the files over to my test VM. In my test VM I could run the script as `.\win-toggle-safe-mode.ps1 -safeModeSwitch off`. I'm guessing this can be ran in az vm repair as  `az vm repair run -g sourceRG -n sourceVM --run-id win-toggle-safe-mode --parameters safeModeSwitch=off --verbose --run-on-repair` going by the syntax of [win-hello-world](https://github.com/Azure/repair-script-library/blob/master/src/windows/win-hello-world.ps1)